src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughWithoutBreakStatements.java

```diff
@@ -0,0 +1,50 @@
+/*
+FallThrough
+checkLastCaseGroup = (default)false
+reliefPattern = (default)falls?[ -]?thr(u|ough)
+
+
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.fallthrough;
+
+public class InputFallThroughWithoutBreakStatements{
+
+
+    static {
+
+    }
+    static {
+        String c = "🎄";
+    }
+    public void fooMethod()
+    {
+        String a = "12🤩";
+        String b = "";
+        if (a == "🤩12🧐🧐") { }
+        char[] s = {'1', '2'};
+        int index = 2;
+        if (doSideEffect() == 1) { }
+        while ((a = "12") != "🧐") {return;}
+        for (; index < s.length && s[index] != 'x'; index++) {}
+        if (a == "12🤣") {} else {System.identityHashCode("a");}
```

Comment:
I don't think all this code matters , please remove all code from input that doesn't matter

Reply:
Done @romani